paper/paper.md

```diff
@@ -0,0 +1,184 @@
+---
+title: 'Pyglmnet: Python implementation of elastic-net regularized generalized linear models'
+tags:
+  - Python
+  - glm
+  - machine-learning
+  - lasso
+  - elastic-net
+  - group-lasso
+authors:
+  - name: Mainak Jas
+    orcid: 0000-0003-0872-7098
```

Comment:
This orcid resolves to someone different, one Adrian Price-Whelan. I think you are https://orcid.org/0000-0002-3199-9027